### PR TITLE
[trivial] remove benchmark dependencies from v3

### PR DESCRIPTION
**Pull request by kwasnick**
Currently, importing `github.com/cep21/circuit/v3` requires transitive dependencies on every circuit-breaking library that is used in the `benchmarking` package, even though they are not actually used by the real library. By creating a separate `go.mod/go.sum` in `benchmarking/`, we can remove these transitive dependencies.

### Process
```bash
cd v3/benchmarking
go mod init
go mod tidy
cd ..
go mod tidy
```

### CODE REVIEW

**Comment by kwasnick** [comment]
@cep21 let me know what you think – seems like just an oversight to include these dependencies in the main v3 package

**Comment by cep21** [comment]
I like this.  Let me think about it today and get back to you.

File: v3/benchmarking/go.mod [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module github.com/kwasnick/circuit/v3/benchmarking
+
+go 1.16
+
+require (
+	github.com/afex/hystrix-go v0.0.0-20180502004556-fa1af6a1f4f5
+	github.com/cep21/circuit/v3 v3.2.0
```
Comment (cep21) [inline v3/benchmarking/go.mod:7]: One problem here, the benchmarking won't change if we update the library code without updating the tag here :(

Comment (kwasnick) [reply v3/benchmarking/go.mod:7]: You're right, that is unfortunate. I'm not 100% sure on how this works, but I believe the `replace` directive with a relative path would work to fix this

```
replace github.com/cep21/circuit/v3 => ../
```

Comment (kwasnick) [reply v3/benchmarking/go.mod:8]: It looks like it works, based on what gets put in go.mod and go.sum with the replace directive https://github.com/cep21/circuit/pull/109/commits/37cce18431d7dcc3c95b70faafc270d07df271b1

**Comment by cep21** [comment]
This appears safe after local testing.  Thanks for the contribution!  I'll merge it in and release a new patch version.

**Pull request merged**